i2c/sample-implementations/GPIO_bit_banging/sample-implementations/Atmel_SAMD2_series/sensirion_i2c_gpio.c

```diff
@@ -129,5 +129,5 @@ uint8_t sensirion_i2c_gpio_SCL_read(void) {
  * @param useconds the sleep time in microseconds
  */
 void sensirion_i2c_gpio_sleep_usec(uint32_t useconds) {
-    // IMPLEMENT
+    /* IMPLEMENT */
```

Comment:
```suggestion
    /* TODO: IMPLEMENT */
```
might help to highlight it or in conjunction with git grep